addons/binding/org.openhab.binding.kostalinverter/ESH-INF/thing/thing-types.xml

```diff
@@ -49,13 +49,1516 @@
+				<channel id="DEVICE_LOCAL_HOMECONSUMPTION_FROM_PV" typeId="DEVICE_LOCAL_HOMECONSUMPTION_FROM_PV" />
+				<channel id="DEVICE_LOCAL_HOMECONSUMPTION_TOTAL" typeId="DEVICE_LOCAL_HOMECONSUMPTION_TOTAL" />
+				<channel id="DEVICE_LOCAL_LIMIT_EVU_ABSOLUTE" typeId="DEVICE_LOCAL_LIMIT_EVU_ABSOLUTE" />
+				<channel id="DEVICE_LOCAL_LIMIT_EVU_RELATIV" typeId="DEVICE_LOCAL_LIMIT_EVU_RELATIV" />
+				<channel id="DEVICE_LOCAL_WORKTIME" typeId="DEVICE_LOCAL_WORKTIME" />
+				<channel id="DEVICE_LOCAL_AC_PHASE_1_CURRENT_AMPERAGE" typeId="DEVICE_LOCAL_AC_PHASE_1_CURRENT_AMPERAGE" />
+				<channel id="DEVICE_LOCAL_AC_PHASE_1_CURRENT_POWER" typeId="DEVICE_LOCAL_AC_PHASE_1_CURRENT_POWER" />
+				<channel id="DEVICE_LOCAL_AC_PHASE_1_CURRENT_VOLTAGE" typeId="DEVICE_LOCAL_AC_PHASE_1_CURRENT_VOLTAGE" />
+				<channel id="DEVICE_LOCAL_AC_PHASE_2_CURRENT_AMPERAGE" typeId="DEVICE_LOCAL_AC_PHASE_2_CURRENT_AMPERAGE" />
+				<channel id="DEVICE_LOCAL_AC_PHASE_2_CURRENT_POWER" typeId="DEVICE_LOCAL_AC_PHASE_2_CURRENT_POWER" />
+				<channel id="DEVICE_LOCAL_AC_PHASE_2_CURRENT_VOLTAGE" typeId="DEVICE_LOCAL_AC_PHASE_2_CURRENT_VOLTAGE" />
+				<channel id="DEVICE_LOCAL_AC_PHASE_3_CURRENT_AMPERAGE" typeId="DEVICE_LOCAL_AC_PHASE_3_CURRENT_AMPERAGE" />
+				<channel id="DEVICE_LOCAL_AC_PHASE_3_CURRENT_POWER" typeId="DEVICE_LOCAL_AC_PHASE_3_CURRENT_POWER" />
+				<channel id="DEVICE_LOCAL_AC_PHASE_3_CURRENT_VOLTAGE" typeId="DEVICE_LOCAL_AC_PHASE_3_CURRENT_VOLTAGE" />
+				<channel id="DEVICE_LOCAL_AC_CURRENT_POWER" typeId="DEVICE_LOCAL_AC_CURRENT_POWER" />
+				<channel id="DEVICE_LOCAL_PVSTRING_1_AMPERAGE" typeId="DEVICE_LOCAL_PVSTRING_1_AMPERAGE" />
+				<channel id="DEVICE_LOCAL_PVSTRING_1_POWER" typeId="DEVICE_LOCAL_PVSTRING_1_POWER" />
+				<channel id="DEVICE_LOCAL_PVSTRING_1_VOLTAGE" typeId="DEVICE_LOCAL_PVSTRING_1_VOLTAGE" />
+				<channel id="DEVICE_LOCAL_PVSTRING_2_AMPERAGE" typeId="DEVICE_LOCAL_PVSTRING_2_AMPERAGE" />
+				<channel id="DEVICE_LOCAL_PVSTRING_2_POWER" typeId="DEVICE_LOCAL_PVSTRING_2_POWER" />
+				<channel id="DEVICE_LOCAL_PVSTRING_2_VOLTAGE" typeId="DEVICE_LOCAL_PVSTRING_2_VOLTAGE" />
+				<channel id="DEVICE_LOCAL_PVSTRING_3_AMPERAGE" typeId="DEVICE_LOCAL_PVSTRING_3_AMPERAGE" />
+				<channel id="DEVICE_LOCAL_PVSTRING_3_POWER" typeId="DEVICE_LOCAL_PVSTRING_3_POWER" />
+				<channel id="DEVICE_LOCAL_PVSTRING_3_VOLTAGE" typeId="DEVICE_LOCAL_PVSTRING_3_VOLTAGE" />
+				<channel id="SCB_EVENT_ERROR_COUNT_MC" typeId="SCB_EVENT_ERROR_COUNT_MC" />
+				<channel id="SCB_EVENT_ERROR_COUNT_SFH" typeId="SCB_EVENT_ERROR_COUNT_SFH" />
+				<channel id="SCB_EVENT_ERROR_COUNT_SCB" typeId="SCB_EVENT_ERROR_COUNT_SCB" />
+				<channel id="SCB_EVENT_WARNING_COUNT_SCB" typeId="SCB_EVENT_WARNING_COUNT_SCB" />
+				<channel id="STATISTIC_AUTARKY_DAY" typeId="STATISTIC_AUTARKY_DAY" />
+				<channel id="STATISTIC_AUTARKY_MONTH" typeId="STATISTIC_AUTARKY_MONTH" />
+				<channel id="STATISTIC_AUTARKY_TOTAL" typeId="STATISTIC_AUT
```

Comment:
I would assume these channels overlap with the once in the original thing. Could you reuse them? And maybe use a similar naming convention.

Reply:
After the refactoring of the channels, I moved the channel definition from the existing thing and updated the definition for the existing thing to match the channels